### PR TITLE
Armorgang

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
Weeeeeeeell.

image

Ended up adding Infra's armors today. I mostly followed the document Hekzder had laid out spare for a couple points in which I believe he was VERY MISINFORMED and MISLED. Didn't give some of the armors such drastic slowdowns, shifted a few numbers around slightly. Mostly lowered flame resistance of the metal armors, if I recall. I also added in the boots/gloves that infra made to go with the jackets, and rebranded the deputy/mayor/sheriff jackets to just be brown/blue/black jackets, because for some reason they were added really weird. Fixed the loot tables to reflect the changes, though I still need to talk to Hekzder a bit more about that because I don't necessarily want to dip my toe into there, so for now it will remain as is.  

All of the armors seem to be functional, all inhands, all worn states **SHOULD** be completely functional.  I'm personally a bit fan of the four special suits, the Radiation suits, Hazmat suits, Firesuits, and the Space Suit. I don't exactly know where the space suit will come in play but I made it basically all three of the previous suits combined to be a jack of all trades at slightly lowered efficiency.  The chinese stealth suit could probably get some more love as soon as Techno's stealthboy PR gets in, I could see someone ripping the code and applying it to the suit.

### CODE REVIEW

**Comment by ProfessorPopoff** [comment]
I'm pretty certain that everything in the screenshot is what was added or modified but i'm not certain. It's been a long morning.

File: mojave/items/clothing/head/helmet.dm [hunk 1]
```diff
@@ -242,23 +249,22 @@
...
-	icon_state = "eyebot"
-	inhand_icon_state = "eyebot"
-	armor = list("melee" = 50, "bullet" = 35, "laser" = 5, "energy" = 5, "bomb" = 30, "bio" = 10, "rad" = 10, "fire" = 50, "acid" = 10)
-	flags_inv = HIDEEARS|HIDEFACE|HIDEHAIR
-	dynamic_hair_suffix = ""
-	dynamic_fhair_suffix = ""
-
 /obj/item/clothing/head/helmet/ms13/metal
 	name = "metal helmet"
 	desc = "A robust but cumbersome metal helmet. Bogs you down a bit in exchange for it's good all around protection. Metal is known to be conductive however."
```
Comment (Hekzder) [inline mojave/items/clothing/head/helmet.dm:254]:
```suggestion
	desc = "A robust metal helmet offering good all around protection. Metal is known to be very conductive however."
```

Given it no longer has slowdown, no need to have the "cumbersome" and "bogs you down" lines.

**Comment by Hekzder** [comment]
All things considered I'm fine with merging the PR in this state. I was originally going to make balance suggestions and such, but it's something I'd need more time and a closer look to get some good ideas on. And having everything already implemented is a super big help in speeding that process along. 

We might as well just have it all implemented now and the exacting balance can just get ironed out at some point before the upcoming combat test.

File: mojave/items/clothing/head/helmet.dm (hunk 1 above)
Comment (ProfessorPopoff) [reply mojave/items/clothing/head/helmet.dm:254]: Truuu!

**Pull request merged**